ROMFS/px4fmu_common/init.d/airframes/CMakeLists.txt

```diff
@@ -131,6 +131,8 @@ px4_add_romfs_files(
 	13010_claire
 	13012_convergence
 	13013_deltaquad
+	13014_vtol_babyshark
```

Comment:
I don’t know if it’s necessary to add it, but it’s a neater format.

Reply:
px4-v2 no flash

Reply:
Thanks for point this, I will see if there other solution for this.